src/main/java/beam/analysis/StatsFactory.java

```diff
@@ -2,11 +2,18 @@
 
 import beam.analysis.cartraveltime.PersonAverageTravelTimeAnalysis;
 import beam.analysis.plots.*;
-import beam.analysis.summary.*;
+import beam.analysis.plots.filterevent.ActivitySimFilterEvent;
```

Comment:
actitivitySim config parameter missing to enable

Reply:
@wrashid added. can be checked in the following lines:
https://github.com/LBNL-UCB-STI/beam/pull/2724/files#diff-4d795c860a10b4727cb9f9da894b3eefR244
https://github.com/LBNL-UCB-STI/beam/pull/2724/files#diff-c8b3324eb5c46ac44755ae58b926a3bdR16